Panic when 'init' exit status != 0

In a stack directory where running a plain `terraform init` exits with a non-zero error and the message `Error: module "BaseNSG": missing required argument "rg_ready"` running `stack init` gives the following panic output:
```
panic: exit status 1

goroutine 35 [running]:
main.initStack.func3(0xc0001e0160, 0xc0001d01a0, 0xc0001de300)
        /drone/src/cmd/stack/init.go:144 +0x74
created by main.initStack
        /drone/src/cmd/stack/init.go:141 +0x14de
```

--- a/cmd/stack/init.go
+++ b/cmd/stack/init.go
@@ -10,6 +10,7 @@ import (
 	"os/exec"
 	"strings"
 	"sync"
+	"syscall"
 
 	"github.com/jlucktay/stack/pkg/common"
 	"github.com/spf13/viper"
@@ -141,7 +142,13 @@ func initStack() {
 	go func() {
 		errWait := cmdInit.Wait()
 		if errWait != nil {
-			panic(errWait)
+			if exitErr, ok := errWait.(*exec.ExitError); ok {
+				if status, ok := exitErr.Sys().(syscall.WaitStatus); ok {
+					log.Printf("Exit status: %d", status.ExitStatus())
+				}
+			} else {
+				panic(errWait)
+			}
 		}
 		wg.Wait()
 		close(chOut)

